api/include/opentelemetry/http/http_client.h

```diff
@@ -0,0 +1,161 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "opentelemetry/nostd/string_view.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace http
+{
+
+enum class HttpResult : uint8_t
+{
+  // Response has been received successfully from target server.
+  HttpResult_OK = 0,
```

Comment:
Please conform the the Google C++ Style Guide. Here's the part for [enums](https://google.github.io/styleguide/cppguide.html#Enumerator_Names).

Reply:
@lalitb FYI https://github.com/open-telemetry/opentelemetry-cpp/blob/master/CONTRIBUTING.md#development

Reply:
Will fix this.